docs/concepts/governance/bal-token.md

```diff
@@ -200,32 +195,29 @@ This chart outlines the allocation amounts. More details below.
   },
   "options": {}
 }
-```
-
 :::
 
 #### Liquidity Providers
 
-Emissions to liquidty providers are decided by a combination of the emissions schedule described above, and [veBAL](./veBAL) voting to determine the rewards that flow to each authorized pool.
+Emissions to liquidity providers are decided by a combination of the emissions schedule described above, and veBAL voting to determine the rewards that flow to each authorized pool.
 
 #### Founders, Options, Advisors, Investors
 
 25M tokens were allocated to founders, stock options, advisors, and investors, all subject to vesting periods.
 
 #### Ecosystem Fund
 
-5M were allocated for the Ecosystem Fund. This fund will be deployed to attract and incentivize strategic partners who will help the Balancer ecosystem grow and thrive. BAL holders will ultimately decide how this fund is used over the coming years.
+5M were allocated for the Ecosystem Fund. This fund will be deployed to attract and incentivize strategic partners who will help the Balancer ecosystem grow and thrive. veBAL holders will ultimately decide how this fund is used over the coming years.
 
-Note that as part of the veBAL conversion, the entire ecosystem fund was transferred into the [DAO multisig](https://etherscan.io/address/0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f)
+Note that as part of the veBAL conversion, the entire ecosystem fund was transferred into the [DAO multisig](https://polygonscan.io/address/0xeE071f4B516F69a1603dA393CdE8e76C40E5Be85).
```

Comment:
This link is broken. Isn't the original correct?

Reply:
Yeah it is.  Was merging in some changes from others here.